### PR TITLE
Update for NUsight's location change (mono repo!)

### PR DESCRIPTION
**Pull request by ysims**
Updates after moving NUsight into the NUbots repository. 

- Getting Started page: Removes instructions to git clone NUsight
- NUsight page: Adds link to Getting Started page, NUsight section
- How We Work page: Removes NUsight from the list of useful repositories

### CODE REVIEW

**Review by tayloryoung6396 — APPROVED** [review]
LGTM

File: src/book/01-team/02-joining/01-how-we-work.mdx
Comment (JosephusPaye) [inline src/book/01-team/02-joining/01-how-we-work.mdx:27]:
```suggestion
- [NUbots](https://github.com/NUbots/NUbots) - monorepo with the C++ codebase that runs on the robots, and NUsight, the web-based debugging environment we used to understand and improve our systems.
```

File: src/book/01-team/02-joining/01-how-we-work.mdx
Comment (JosephusPaye) [inline src/book/01-team/02-joining/01-how-we-work.mdx:28]:
```suggestion
- [NUbook](https://github.com/NUbots/NUbook) - our team handbook, which you are viewing now! Built with Gatsby and deployed using Netlify.
```

**Review by JosephusPaye — APPROVED** [review]
Some suggestions

**Comment by JosephusPaye** [comment]
Meant to request changes, but GitHub won't allow me to change my review. Oh well.

File: src/book/01-team/02-joining/01-how-we-work.mdx
Comment (TrentHouliston) [reply src/book/01-team/02-joining/01-how-we-work.mdx:27]: I was also going to suggest this!

**Review by TrentHouliston — DISMISSED** [review]

**Review by KipHamiltons — APPROVED** [review]
After merging the `.buildkite`s for NUbots and NUsight2, is there anything else which needs to be done?

**Comment by TrentHouliston** [comment]
You can also swap to github actions, there is a PR that does NUbots and NUsight2 with github actions. That way you don't have to work out how to merge the two buildkites

**Comment by KipHamiltons** [comment]
I'll take that as a no :p

**Comment by ysims** [comment]
@TrentHouliston approval? 😄

**Comment by KipHamiltons** [comment]
dismiss @TrentHouliston 's review on this one too and merge it ahah

**Pull request merged**
